src/Repository/YesplanEventRepository.php

```diff
@@ -91,18 +93,23 @@ public function findNewProductionOnlineEvents(): array
      */
     public function findNewProductionOnlineIncludingGratisandExternEvents(): array
     {
-        //get all events with productiononline = 1 not already created in Asana
+        // get all events with productiononline = 1 not already created in Asana
```

Comment:
This comment (on the return value) does not match the one in the PHPDoc block for the function. Update the PHPDoc block and clean up this comment.